components/explore/ExploreSort.vue

```diff
@@ -71,16 +71,23 @@ const options = computed(() => {
 })
 
 function removeDuplicateSortKeys(options: string[]) {
+  const uniqueOptions = {}
+
   if (!Array.isArray(options)) {
     return []
   }
-  const uniqueSortOptions = options.reduce((dict, option) => {
-    const [sortKey, modifier] = option.split('_')
-    return { ...dict, [sortKey]: modifier }
-  }, {})
-  return Object.entries(uniqueSortOptions).map(
-    ([sortKey, modifier]) => `${sortKey}_${modifier}`
-  )
+
+  options.forEach((option) => {
+    const opt = option.split('_')
+    const identifier = opt[0]
+    const sort = opt[1]
+
+    uniqueOptions[identifier] = sort
+  })
+
+  return Object.keys(uniqueOptions).map((identifier) => {
+    return `${identifier}_${uniqueOptions[identifier]}`
+  })
```

Comment:
```suggestion

  return Array.from(new Set(options))
```

How about removing duplicate item in this way? It would be more simple and easy to read I guess

Reply:
no. that will not do...
for example, sorting by recently created is ```blockNumber_DESC``` and sort by oldest is ```blockNumber_ASC```
the unique bit is ```blockNumber``` and only one option should be allowed
by mapping the array into object, we keep only the latest selected sort option, which makes sense

![image](https://user-images.githubusercontent.com/22791238/224703140-57d6be82-eb42-4bb7-8c51-a2f52b6bb5f4.png)

Reply:
oh, nvm. I should have read the original issue description : )